src/index.html

```diff
@@ -2,16 +2,141 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldsett">
+        <legend>Personal information</legend>
+        <label class="form-field">
+           Surname:
+          <input type="text"
```

Comment:
Please fix this for all case
![image](https://user-images.githubusercontent.com/54936044/219342882-4c729e16-d73b-4aaa-b1b5-561cb32508a5.png)